.github/workflows/golangci-lint.yaml

```diff
@@ -17,4 +17,4 @@ jobs:
       uses: golangci/golangci-lint-action@v3
       with:
         version: v1.55.2
```

Comment:
All checks are version dependent, the current version of lint has no warnings in CI, you should upgrade this version and fix the warnings.

Reply:
> All checks are version dependent, the current version of lint has no warnings in CI, you should upgrade this version and fix the warnings.

Okay, will look into this

Reply:
I've bumped up the version to 1.58 as per your suggestion in the new commit